source/release-notes.rst

```diff
@@ -15,6 +15,123 @@ Releases
+* Fix for Bluetooth crash issue.
+
+Existing Features
+-----------------
+
+* Debian package release with Adaptive Installation Script support
+* Support android reboot in cic-monitor service
+* Klocwork issue fixes
+* Support for multi window display
+* Camera back and front switching implementation
+* HDMI display support
+* Mass Storage USB 2.0 and 3.x devices is supported
+* Adb over WIFI and Ethernet is supported
+* Audio playback over USB Headset and HDMI are supported
+* Wi-Fi 802.11 a/b/g/n/ac and Bluetooth 4.2
+* Wired Ethernet support
+* Video Codec support on celadon
+* Sdcardfs filesystem support enabled
+* Audio decoders supported: MP3, AAC-LC, AAC-ELD, HEAAC, HEAAC-V2, VORBIS, OPUS, MIDI, FLAC, PCM/WAV
+* SDHCI host controller is enabled
+* eMMC / SATA / NVMe storage media supported
+* Generic storage HAL supported
+* Vendor WiFi hal integrated
+* Battery Indicator enabled
+* From storage, Sdcardfs filesystem support, eMMC 5.1 controller driver support, eMMC Storage support enabled
+* Mesa upgraded to 19.3.3
+* Following features added on AOSP side :
+    * Support host shortcut for android apps
+    * Support copy and paste feature between AIC and Host
+    * Support multi-user data isolation
+    * Add cic monitor service to support shutdown and reboot
+
+Known Issues
+-------------
+* Unable to install graphics apk (Less Memory)
+* Multi-camera application crashes after hotplug
+* Unable to control volume using 3.5mm headset volume buttons.
+* Stability KPI-2 Fails
+* Unable to control android volume using USB headset volume buttons.
+* Getting Error "transfer error: secure_mkdirs failed: Operation not permitted" while running cts
+* Memory allocation doesn't show for graphics('EGL mtrack or Gfx).
+* Unable to capture photo and video after hot-plug.
+* Camera flip icon is displayed after hotplug.
+* After connecting Wi-Fi in Android Container adb over emulator does not work and stability KPI-1 is blocked as WIFI on/off will disconnect adb over WIFI
+* High klocwork error issues in Video domain
+* High klocwork error issues in Energy Management domain
+* GPU Overdraw options having Blue Screen
+* App issue - some app display disorder in multi-window mode
+* App issue - some app flick in multi-window mode
+* Don’t Suspend / press power button on cml host. Device cant be recovered until rebooted. WA : Change power settings to "S3 Legacy standby" instead of "modern standby"
+* Trusty is not supported due to conflict in BIOS.
+* On Security enabled cic, adb install / playstore download are the only options for 3rd party apk installation.
+* Adb goes offline frequently on adb-ethernet.
+* To modify cpu cores usage : After Container start up run the command "sudo docker update --cpuset-cpus="0-11" android0_PID" , where android0_PID is the Docker PID of android instance.
+
+Validation Results
+------------------
+
+This |C| build has been validated on CML NUC - NUC10i7FN* in the following function domains, for both
```

Comment:
what does this "|C|" mean?

Reply:
@kwang13 |C| is a Sphinx directive to substitute the enclosed text with the desired string. In this case, it will be replaced by 'Celadon'. The other substitutions can be found in [this file](https://github.com/projectceladon/celadon-documentation/blob/master/source/substitutions.txt). FYI.